### PR TITLE
docs: update Articles and Applications

### PR DESCRIPTION
**Pull request by sabesansathananthan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +80,8 @@ A list of UI components built with Material-UI Design.
 
 A list of mature apps built with Material-UI Design.
 
+- [Material-ui Medium Blog](https://github.com/sabesansathananthan/material-ui-medium-blog)
+- [React Github Repo Cards](https://github.com/sabesansathananthan/react-github-repo-cards)
```
Comment (nadunindunil) [inline README.md:84]: This goes under the components section
and please add a description after '-' sign

eg: [bethesda.net](https://bethesda.net/) - The official site for Bethesda, publisher of Fallout, DOOM, Dishonored, Skyrim, Wolfenstein, The Elder Scrolls, more.

File: README.md [hunk 2]
```diff
@@ -77,6 +80,8 @@ A list of UI components built with Material-UI Design.
 
 A list of mature apps built with Material-UI Design.
 
+- [Material-ui Medium Blog](https://github.com/sabesansathananthan/material-ui-medium-blog)
```
Comment (nadunindunil) [inline README.md:83]: Should be changed ad Material-**UI**
and please add a description after '-' sign

eg: bethesda.net - The official site for Bethesda, publisher of Fallout, DOOM, Dishonored, Skyrim, Wolfenstein, The Elder Scrolls, more.

**Review by nadunindunil — CHANGES_REQUESTED** [review]
Requested a few changes

File: README.md (hunk 1 above)
Comment (sabesansathananthan) [reply README.md:84]: Sure @nadunindunil ayya.

**Review by nadunindunil — APPROVED** [review]

**Comment by nadunindunil** [comment]
@sabesansathananthan this doesn't work - https://react-github-repo-cards.vercel.app/
can you please check?

**Pull request merged**